articles/service-fabric/service-fabric-cicd-your-linux-java-application-with-jenkins.md

```diff
@@ -65,29 +69,27 @@ bash Scripts/install.sh
 ```
 This installs a Jenkins container on the cluster, and can be monitored by using the Service Fabric Explorer.
 
-### Steps
-1. From your browser, go to ``http://PublicIPorFQDN:8081``. It provides the path of the initial admin password required to sign in. You can continue to use Jenkins as an admin user. Or you can create and change the user, after you sign in with the initial admin account.
-
    > [!NOTE]
-   > Ensure that the 8081 port is specified as the application endpoint port while you are creating the application (and the port is open in the cluster).
+   > It may take a couple of minutes for the Jenkins image to be downloaded on the cluster.
    >
 
-2. Get the container instance ID by using ``docker ps -a``.
-3. Secure Shell (SSH) sign in to the container, and paste the path you were shown on the Jenkins portal. For example, if in the portal it shows the path `PATH_TO_INITIAL_ADMIN_PASSWORD`, run the following:
+### Steps
+1. From your browser, go to ``http://PublicIPorFQDN:8081``. It provides the path of the initial admin password required to sign in. 
+2. Look at the Service Fabric Explorer to determine on which node the Jenkins container is running. Secure Shell (SSH) sign in to this node.
+```sh
+ssh user@PublicIPorFQDN -p [port]
+``` 
+3. Get the container instance ID by using ``docker ps -a``.
+4. Secure Shell (SSH) sign in to the container, and paste the path you were shown on the Jenkins portal. For example, if in the portal it shows the path `PATH_TO_INITIAL_ADMIN_PASSWORD`, run the following:
 
   ```sh
   docker exec -t -i [first-four-digits-of-container-ID] /bin/bash   # This takes you inside Docker shell
-  cat PATH_TO_INITIAL_ADMIN_PASSWORD
   ```
-
-4. Set up GitHub to work with Jenkins, by using the steps mentioned in [Generating a new SSH key and adding it to the SSH agent](https://help.github.com/articles/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent/).
-	* Use the instructions provided by GitHub to generate the SSH key, and to add the SSH key to the GitHub account that is hosting your repository.
-	* Run the commands mentioned in the preceding link in the Jenkins Docker shell (and not on your host).
-	* To sign in to the Jenkins shell from your host, use the following command:
-
   ```sh
-  docker exec -t -i [first-four-digits-of-container-ID] /bin/bash
+  cat PATH_TO_INITIAL_ADMIN_PASSWORD # This displays the pasword value
   ```
+5. On the Jenkins Gettting Started page, choose the Select plugins to install option, select any desired plugins, and click install.
```

Comment:
For our use-case, we can ask the user to select **None** from the select-plugins options. If the user unknowingly or without enough context tries installing some or all plugins, that would greatly deteriorate the experience.

Reply:
Updated.